google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -3540,6 +3540,10 @@ HmacKeyMetadata updateHmacKeyState(
   /**
    * Gets the IAM policy for the provided bucket.
    *
+   * <p>It's possible for bindings to be empty and instead have permissions inherited through
+   * Project or Organization IAM Policies. To prevent corrupting policies when performing a {@code
```

Comment:
"To prevent corrupting policies when you update an IAM policy with {@code Storage.setIamPolicy, the ETAG value..."